tests/test_audio.py

```diff
@@ -51,28 +52,52 @@ def test_get_integrated_lufs():
         i = get_integrated_lufs(af)
         assert i == li
 
+def change_format_and_subtype(audio_path):
+    audio, sr = sf.read(audio_path)
+
+    formats = ['WAV', 'FLAC']
+    _format = random.choice(formats)
+
+    subtypes = sf.available_subtypes(_format)
+    accepted_subtypes = ['PCM_16', 'PCM_32', 'PCM_24', 'FLOAT', 'DOUBLE']
+    subtypes = [s for s in subtypes.keys() if s in accepted_subtypes]
+    _subtype = random.choice(subtypes)
+    
+    sf.write(audio_path, audio, sr, subtype=_subtype, format=_format)
```

Comment:
If I'm not mistaken this test doesn't guarnatee the format and subtype will be *changed*, since there's a non-zero probability the original format and subtype will be randomly selected. Can we remove the original format and subtype from possible options, so that we guarantee the format and type are changed?

Should be able to do something like:
```
audio, sr = sf.read(audio_path)
audio_info = soundfile.info(audio_path)
formats = ['WAV', 'FLAC'] - audio_info.format
_format = random.choice(formats)
subtypes = sf.available_subtypes(_format) - audio_info.subtype
```

right?

Reply:
Fixed